src/test/java/org/graylog/integrations/notifications/types/SlackEventNotificationConfigTest.java

```diff
@@ -30,4 +32,39 @@ public void validate_failsAndReturnsAnError_whenWebhookUrlIsInvalid() {
         Map errors = result.getErrors();
         assertThat(errors).size().isGreaterThan(0);
     }
+
+    @Test
+    public void validate_messageBacklog() {
```

Comment:
Generally speaking, a test case should only be testing one behavior at a time.  It would probably be good to split this into three separate tests for the three different conditions being tested.  

As mentioned earlier in the PR, I don't think that we should be enforcing an upper limit on `backlogSize` so that eliminates one test.  However, since zero is intended to have special meaning, you should have a test verifying that zero is a valid value.